src/vignette/http/routes.clj

```diff
@@ -203,3 +203,50 @@
     (clojure.string/replace (:image-type route-params)
                             #"^\/(.*)"
                             "$1")))
+
+(defn route->image-type
```

Comment:
Yeah. I was going to save that for later if it's ok. We have the same issue in the legacy routes namespace.